python-wrapper/LICENSE

```diff
@@ -0,0 +1 @@
+../LICENSE
```

Comment:
This is a symlink to the base repo's license. Useful so we keep the same license in the pip releases without needing to remember to edit both if edits are necessary